File: APIRollTheDice-Backend/src/main/java/com/example/APIRollTheDice/Model/DTO/GameDTO/TemplateDTO/TemplateDTO.java
package com.example.APIRollTheDice.Model.DTO.GameDTO.TemplateDTO;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class TemplateDTO {
    @JsonProperty("Id")
    private Long id;
    @JsonProperty("Name")
    private String name;
    @JsonProperty("IdGameBundle")
    private Long idGameBundle;
    @JsonProperty("IdTemplateFieldList")
    private List<Long> idTemplateFieldList;

    public TemplateDTO() {
    }

    public TemplateDTO(Long id, String name, Long idGameBundle, List<Long> idTemplateFieldList) {
        this.id = id;
        this.name = name;
        this.idGameBundle = idGameBundle;
        this.idTemplateFieldList = idTemplateFieldList;
    }


    public Long getId() {
        return id;
    }

    public void setId(long id) {
        this.id = id;
    }

    public Long getIdGameBundle() {
        return idGameBundle;
    }

    public void setIdGameBundle(Long idGameBundle) {
        this.idGameBundle = idGameBundle;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<Long> getIdTemplateFieldList() {
        return idTemplateFieldList;
    }

    public void setIdTemplateFieldList(List<Long> idTemplateFieldList) {
        this.idTemplateFieldList = idTemplateFieldList;
    }
}

File: RollTheDice/Assets/_Project/API/Model/DTO/GameDTO/TemplateDTO/TemplateDTO.cs
using System.Collections.Generic;

namespace Assets._Project.API.Model.DTO.GameDTO.TemplateDTO
{
    public class TemplateDTO
    {
        public long Id { get; set;}
        public string Name { get; set;}
        public long IdGameBundle { get; set;}
        public List<long> IdTemplateFieldList { get; set;}

        public TemplateDTO(){}
        public TemplateDTO(long id, string name, long idGameBundle, List<long>idTemplateFieldList )
        {
            Id = id;
            Name = name;
            IdGameBundle = idGameBundle;
            IdTemplateFieldList = idTemplateFieldList;
     
        }

    }
}